test/cli_volume_test.go

```diff
@@ -350,4 +350,29 @@ func (suite *PouchVolumeSuite) TestVolumeListOptions(c *check.C) {
 			break
 		}
 	}
+
+	// test filter options
+	volumeName4 := "volume_" + funcname + "4"
+	command.PouchRun("volume", "create", "--name", volumeName4, "--driver", "tmpfs", "--label", "test=foo").Assert(c, icmd.Success)
+	defer command.PouchRun("volume", "rm", volumeName4)
+
+	// test name, label, driver filter
+	filterArgs := []string{
+		"name=" + volumeName4,
+		"label=test",
+		"driver=tmpfs",
```

Comment:
how about add some `and` case, like `--lable=test --name=a --driver=tmpfs`

Reply:
Agree.